disnake/ui/button.py

```diff
@@ -284,6 +326,12 @@ def button(
 
     Parameters
     ----------
+    cls: Type[:class:`Button`]
```

Comment:
Should clarify that if this parameter is specified, the following documented parameters (`label`, `custom_id`, ...) aren't applicable anymore

Reply:
Should I include that in the description of the `cls` parameter, or maybe before the list of parameters?